### PR TITLE
feat: included up-to-date information

### PR DESCRIPTION
**Pull request by Aidasvenc**
Some sections of current GHO documentation date back to before the stablecoin was launched. Therefore, an update of GHO documentation was performed to reflect the current state of the stablecoin as of May 2024.

List of changes:
- Changed some references like "will be" to a present tense.
- Clarified unclear statements.
- Updated the `Facilitators` section and added a subsection explaining GHO stability module.
- Clarified Governance structure and included an explanation covering GHO Liquidity Committee and GHO Stewards.

**Disclaimer:** these changes firstly need to be approved internally by Llama Risk team before escalating and merging them into the base repository.

### CODE REVIEW

File: docs/concepts/faq.md [hunk 1]
```diff
@@ -33,21 +33,21 @@ Unlike many stablecoins, the oracle price for GHO is fixed. Decentralized stable
 
 ## Which assets can be used as collateral to borrow GHO?
 
-Assets that are available in the Aave Protocol can be used to back GHO. Initially, the Ethereum V3 pool will be the first facilitator to launch because of V3’s extensive risk-mitigation features, including e-mode, isolation mode, and supply caps.
+Assets that are available in the Aave Protocol Lending Pool can be used to back GHO. The Ethereum V3 pool was first facilitator to launch and support minting (borrowing) GHO. It was possible because of V3’s extensive risk-mitigation features, including e-mode, isolation mode, and supply caps.
```
Comment (0xValJohn) [inline docs/concepts/faq.md:36]: missing "the" after "was"

File: docs/concepts/faq.md [hunk 2]
```diff
@@ -33,21 +33,21 @@ Unlike many stablecoins, the oracle price for GHO is fixed. Decentralized stable
...
 
 ## Who manages the GHO supply?
 
-The Aave DAO will manage the supply of GHO, the interest rates and determine risk parameters.
+The Aave DAO and different Governance assigned entities manage the supply of GHO, the interest rates and determine risk parameters.
 
 ## What is a Facilitator and what does it mean for GHO?
 
-GHO introduces the concept of [Facilitators](../concepts/how-gho-works/gho-facilitators.md). A Facilitator (e.g., a protocol, an entity, etc.) has the ability to trustlessly mint (and burn) GHO tokens. To be added as a Facilitator they would have to be approved by Aave Governance. Various Facilitators will be able to apply different strategies to their generation of GHO.
+GHO introduces the concept of [Facilitators](../concepts/how-gho-works/gho-facilitators.md). A Facilitator (e.g., a protocol, an entity, etc.) has the ability to trustlessly mint (and burn) GHO tokens. To be added as a Facilitator they would have to be approved by Aave Governance. Various Facilitators are able to apply different strategies to their generation of GHO.
```
Comment (0xValJohn) [inline docs/concepts/faq.md:44]: change verbe tense here, "would have"

File: docs/concepts/how-gho-works/gho-facilitators.md [hunk 3]
```diff
@@ -9,31 +9,41 @@ GHO introduces the concept of Facilitators. A Facilitator can trustlessly mint a
...
 The [Aave Protocol](https://aave.com/) is a liquidity protocol available on Ethereum and various other L1 and L2 networks. Aave allows users to supply a range of assets and borrow against them while simultaneously earning yield, as well as participating in liquidations. The Aave Protocol operates on an overcollateralized model. Accordingly, GHO is overcollateralized as well.
 
 ### FlashMinter
 
-The FlashMinter Facilitator is an entity that enables [FlashMinting](../fundamental-concepts/flashmint.md). FlashMinting is especially important for GHO as it will help to facilitate arbitrage, provide instant liquidity, and have the ability to liquidate users.
+The FlashMinter Facilitator is an entity that enables [FlashMinting](../fundamental-concepts/flashmint.md). FlashMinting is especially important for GHO as it helps to facilitate arbitrage, provide instant liquidity, and have the ability to liquidate unhealthy GHO debt.
 
 As FlashMinting provides the same functionality as the current [flashloan](https://docs.aave.com/developers/guides/flash-loans) standard, it works very much the same (e.g. everything must be returned and there will be a fee).
 
+### GHO Stability Module
```
Comment (0xValJohn) [inline docs/concepts/how-gho-works/gho-facilitators.md:26]: change to be inline with the hackmd writeup

File: docs/concepts/how-gho-works/gho-facilitators.md [hunk 4]
```diff
@@ -9,31 +9,41 @@ GHO introduces the concept of Facilitators. A Facilitator can trustlessly mint a
...
+The GSM Facilitator introduces the concept of Price Strategies which provide the ability to adjust the pricing ratio between GHO and the exogenous asset, based on different strategies. Pricing strategies oversee the calculation of the price ratio, and:
+- Can be fixed.
+- Can be dynamic, based on price oracles and markets, linear curves, stableswap curves, etc.
+
+Currently, GSM implements a Fixed Pricing Strategy where USDT and USDC can be used to mint GHO. GSM market parameters can be changed by Aave's Governance. GHO Stability Module's interface is supervised by TokenLogic, an Aave DAO Service Provider, and can be accessed in a [dedicated website](https://app.gsm.tokenlogic.xyz/). 
+
 ### Become a Facilitator
 
-Each Facilitator must be approved by [Aave Governance](https://governance.aave.com/). Aave Governance will be able to determine and assign a Facilitator a specific Bucket capacity to bootstrap GHO liquidity and the GHO market.
+Each Facilitator must be approved by [Aave Governance](https://governance.aave.com/). Before approving it, Aave Governance needs to determine and assign a Facilitator a specific Bucket capacity to bootstrap GHO liquidity and the GHO market.
```
Comment (0xValJohn) [inline docs/concepts/how-gho-works/gho-facilitators.md:38]: before approval (remove "it")

File: docs/concepts/how-gho-works/gho-governance.md [hunk 5]
```diff
@@ -18,24 +18,57 @@ The first [GHO Development Update](https://governance.aave.com/t/gho-development
...
+Aave Governance has control over the entities that are added and removed as [Facilitators](./gho-facilitators.md). Every Facilitator is carefully audited and reviewed by Aave Governance before receiving mint permissions. In addition, each Facilitator has a governance-defined limit on how much GHO it can mint.
 
 By carefully approving each Facilitator, Aave Governance can manage exposure to different [strategies](./gho-facilitators.md#facilitator-strategies). Aave Governance may want to attract varied Facilitators in several different ways. This could also include strategies for non-Aave users. If non-Aave Protocol Facilitators are approved by Aave Governance, the possibilities and use cases could be endless (e.g., use within payment applications and e-commerce solutions, purchases of GHO on decentralized exchanges, and use as a store of value during market volatility).
 
-### Interest and Discount Rates
+### GHO Liquidity Committee
```
Comment (0xValJohn) [inline docs/concepts/how-gho-works/gho-governance.md:33]: change to be inline with hackmd writeup

File: docs/concepts/how-gho-works/gho-governance.md [hunk 6]
```diff
@@ -18,24 +18,57 @@ The first [GHO Development Update](https://governance.aave.com/t/gho-development
...
+With many liquidity pools to be created and rewards distributed across, it is important the DAO can swiftly increase the GHO Borrow Cap to mitigate GHO trading above $1. The GHO Stewards will be able to increase the GHO Borrow Cap swiftly to mitigate GHO trading above peg. The GHO Stewards can increase the GHO Borrow Cap up to a threshold of 50M units to a total borrow cap of 100M.
 
-Aave Governance can periodically set the interest rates of GHO based on supply and demand models. Both the interest rate and the discount rate are configurable by Aave Governance.
+The Borrow Rate is to be adjusted gradually to enable the ecosystem to expand in a safe way. If the trailing 30 day average price of GHO stays outside a $0.995 - $1.005 price range, the GHO Stewards are able adjust the Borrow Rate no more than 500bps per 2 day period, up to a maximum 25% APR.
 
-For more information, please see the [Interest and Discount Rates](interest-rate-discount-model.md) page.
+GHO Stewards consists of members from Growth (ACI), Risk (ChaosLabs) and Finance (TokenLogic + karpatkey) Service Providers and utilise a 3 of 4 multi-sig.
```
Comment (0xValJohn) [inline docs/concepts/how-gho-works/gho-governance.md:70]: need to check if it's still the case

File: docs/concepts/how-gho-works/gho-implementation.md [hunk 7]
```diff
@@ -62,24 +60,26 @@ If GHO is not supplied, there are no suppliers to pay and therefore GHO does not
...
 
-GHO works in the same way as assets already on the Aave Protocol. If the value of users’ collateral drops due to market conditions and their health factors decrease, liquidations can occur.
+## Borrow cap
 
-### Flashloan
+The total amount of GHO that can be minted is limited by a borrow cap that can be and is regularly adjusted by Aave's Governance.
 
-GHO can be [FlashMinted](../fundamental-concepts/flashmint.md) via the FlashMinter Facilitator. This provides the same functionality as a flashloan in the current Aave pool; however, it is implemented differently from how flashloans are integrated today.
+After the launch, GHO stablecoin struggled to keep its peg. In order to counter these issues Aave's governance began closely monitoring GHO's market parameters and performance. Ever since then, Aave's Governance has taken a conservative approach to the borrow cap increases opening the borrow cap step-by-step. Borrow cap and other GHO metrics can be monitored in TokenLogic [GHO analytics dashboard](https://aave.tokenlogic.xyz/gho).
```
Comment (0xValJohn) [inline docs/concepts/how-gho-works/gho-implementation.md:77]: Not sure this is relevant to docs. remove

**Review by 0xValJohn — CHANGES_REQUESTED** [review]

File: docs/concepts/how-gho-works/gho-facilitators.md (hunk 3 above)
Comment (Aidasvenc) [reply docs/concepts/how-gho-works/gho-facilitators.md:26]: Isn't this inline with other subsections? The subsection `Current facilitators` comprises of: `Aave v3 Ethereum Pool`, `FlashMinter`, `GHO Stability Module`.

File: docs/concepts/how-gho-works/gho-governance.md (hunk 6 above)
Comment (Aidasvenc) [reply docs/concepts/how-gho-works/gho-governance.md:70]: Yep, it is the current structure as per https://governance.aave.com/t/arfc-gho-stewards/16466

**Pull request merged**